src/playlist-loader.js

```diff
@@ -319,6 +319,16 @@ export default class PlaylistLoader extends EventTarget {
       throw new Error('Cannot switch media playlist from ' + this.state);
     }
 
+    window.clearTimeout(this.mediaUpdateTimeout);
+
+    if (isFinalRendition) {
+      const delay = (playlist.targetDuration / 2) * 1000 || 5 * 1000;
```

Comment:
I don't think we will encounter this because of when this is called with `isFinalRendition === true`, `playlist` is an object, but we support `playlist` being a string as you can see just below this, so it may be better to move this block below the `string` check

Reply:
👍 updated